digital_benchmark/instagram_benchmark/models.py

```diff
@@ -3,7 +3,7 @@
 # Create your models here.
 
 class InstagramUser(models.Model):
```

Comment:
Do we need to add a FK reference to User?

Reply:
yes, we do need this. I have added this in coming PR